vespa-cloud/cord-19-search/scripts/convert-to-json.py

```diff
@@ -84,9 +93,13 @@ def fall_back_authors(authors):
       lastname = parts[0]
     else:
       lastname,firstname = parts[0],parts[1]
+    name = None
```

Comment:
Seems like `lastname` can be set by itself?
```suggestion
    name = lastname
```